app/tests/unit/server/middleware/attachmentDownloadRouter.test.tsx

```diff
@@ -0,0 +1,114 @@
+/**
+ * @jest-environment node
+ */
+import {
+  attachmentDownloadRouter,
+  handleDownload,
+} from "server/middleware/attachmentDownloadRouter";
+
+const performQuery = require("../../../../server/postgraphile/graphql");
```

Comment:
I ~* think you might be able to shorten the require path to just `server/postgraphile/graphql`?